oak_dice/src/evidence.rs

```diff
@@ -124,10 +124,10 @@ pub struct ApplicationKeys {
 
 static_assertions::assert_eq_size!([u8; 2176], ApplicationKeys);
 
-/// Wrapper for passing DICE info from the Restricted Kernel to the application.
+/// Wrapper for passing the attestation evidence from the Restricted Kernel to the application.
 #[derive(AsBytes, FromBytes)]
 #[repr(C)]
-pub struct RestrictedKernelDiceData {
+pub struct Evidence {
```

Comment:
For this to become `Evidence`, the `ApplicationKeys` need to be refactored, as it currenlty contains private keys as well. So the approach would be to split `ApplicationKeys` into two structs: `ApplicationKeys` that only contains public keys and `ApplicationPrivateKeys` that contain the private keys.

Reply:
good point, in the protos it's also public keys only.